unit/unitTestsLensPrimitives.ml

```diff
@@ -294,10 +318,17 @@ let test_join_lens_2 n test_ctx =
     let res = Query.filter (Query.lt 40 << Query.col "b") (lens_get l3 ()) in
     LensTestHelpers.print_verbose test_ctx (string_of_value (lens_get l3 ())); 
     LensTestHelpers.print_verbose test_ctx (string_of_value res);
+<<<<<<< HEAD
```

Comment:
Looks like an unresolved merge conflict.

Reply:
This should be fixed.